ambry-tools/src/main/java/com.github.ambry/store/StoreCopier.java

```diff
@@ -247,8 +247,8 @@ public void close() throws IOException {
         }
       }
       token = findInfo.getFindToken();
-      logger.info("[{}] [{}] {}% copied", Thread.currentThread().getName(), storeId,
-          df.format(token.getBytesRead() * 100.0 / src.getSizeInBytes()));
+      double percentBytesRead = src.getSizeInBytes() > 0L ? token.getBytesRead() * 100.0 / src.getSizeInBytes() : 100.0;
```

Comment:
I think the problem here is that `token.getBytesRead()` returns `0` - not that `src.getSizeInBytes()` is 0 sized (which it never is because the log segment headers occupy 18 bytes). 

A better check is what @lightningrob added recently - `src.isEmpty()`

```
double percentBytesRead = src.isEmpty() ? 100.0 : token.getBytesRead() * 100.0 / src.getSizeInBytes() ;
```

Reply:
fixed